src/Symfony/Component/DependencyInjection/Compiler/PassConfig.php

```diff
@@ -154,7 +154,7 @@ public function getRemovingPasses()
     /**
      * Gets all passes for the Merge pass.
      *
-     * @return array An array of passes
+     * @return CompilerPassInterface An array of passes
```

Comment:
You mean they should be changed to `CompilerPassInterface[]`? Because all other passes are array, just the merge pass is not.